pkg/auth/csp/tanzu.go

```diff
@@ -73,6 +83,31 @@ func WithOrgID(orgID string) LoginOption {
 	}
 }
 
+// WithListenPort specifies a TCP listen port on localhost, which will be used for the redirect_uri and to handle the
+// authorization code callback. By default, a random high port will be chosen which requires the authorization server
+// to support wildcard port numbers as described by https://tools.ietf.org/html/rfc8252#section-7.3:
+func WithListenPort(port uint16) LoginOption {
```

Comment:
Nit: This is not used, but seems reasonable to include. If you do, I suggest naming being more consistent in the naming in fn cames and comments (ListenPort vs ListenerPort)

I might also add something like:
Being able to designate the listener port might be advantages under some circumstances (e.g. for determining what to port-forward from the host where the web browser is available)

Reply:
Updated the method name to `WithListenerPort` and updated the comment for both `WithListenerPort` and `WithListenerPortFromEnv`. Thanks